Makefile

```diff
@@ -13,7 +13,14 @@ install-deps:
 	apt-get install --no-install-recommends -y build-essential clang protobuf-compiler ocl-icd-opencl-dev aria2 cmake
 
 install-lint-tools:
-	RUSTFLAGS="-Cstrip=symbols" cargo install --locked taplo-cli cargo-audit cargo-spellcheck cargo-udeps
+	-which taplo && cargo quickinstall taplo-cli
+	-which cargo-audit && cargo quickinstall cargo-audit
```

Comment:
Should this be || ?

Reply:
Oh well, I'll fix this and not write a single line of code today because my brain got a Guru Meditation